package.json

```diff
@@ -32,7 +32,8 @@
     "./lib/cjs/util/joinPath.js": "./lib/cjs/util/joinPath.web.js",
     "./lib/cjs/util/multipart.js": "./lib/cjs/util/multipart.web.js",
     "./lib/cjs/util/request.js": "./lib/cjs/util/request.web.js",
-    "./lib/cjs/util/webSocket.js": "./lib/cjs/util/webSocket.web.js"
+    "./lib/cjs/util/webSocket.js": "./lib/cjs/util/webSocket.web.js",
+    "./lib/cjs/util/fs.js": "./lib/cjs/util/fs.web.js"
```

Comment:
As `fs` in an internal module and we are not really providing utility for file IO, we should rename this so something closer. An example could be `util/parseCSV.js`

Reply:
updated